sunpy/net/dataretriever/tests/test_noaa.py

```diff
@@ -8,9 +10,51 @@
 from sunpy.net.fido_factory import UnifiedResponse
 from sunpy.net import Fido
 from sunpy.net import attrs as a
+
 LCClient = noaa.NOAAIndicesClient()
 
 
+def create_mock_unified_object(start_date, end_date):
+    '''
+    Creation of a QueryResponse object, and prefill some
+    downloaded data from noaa.NOAAIndicesClient().fetch(Time('20 ..)
+    '''
+    # Create a mock QueryResponse object
+    map_ = {
+        'Time_start': parse_time(start_date),
+        'Time_end':  parse_time(end_date),
+        'source': 'sdic'
+    }
+
+    resp = QueryResponse.create(map_, LCClient._get_default_uri())
+    # Attach the client with the QueryResponse
+    resp.client = 'noaa-indices'
+    return resp
+
+
+@pytest.mark.remote_data
+def test_fetch_working():
+    '''
+    Tests if the online server for noaa is working.
+    Uses the url : ftp://ftp.swpc.noaa.gov/pub/weekly/RecentIndices.txt
+    '''
+    qr1 = LCClient.search(Time('2012/10/4', '2012/10/6'),
+                          Instrument('noaa-indices'))
+
+    # Mock UnifiedResponse object
```

Comment:
This now just a QueryResponse?

Reply:
Commenting changes, should have removed.